dpnp/tests/test_sycl_queue.py

```diff
@@ -1453,6 +1453,24 @@ def test_choose(device):
     assert_sycl_queue_equal(result.sycl_queue, chc.sycl_queue)
 
 
+@pytest.mark.parametrize("device", valid_dev, ids=dev_ids)
+@pytest.mark.parametrize("left", [None, dpnp.array(-1.0)])
+@pytest.mark.parametrize("right", [None, dpnp.array(99.0)])
+@pytest.mark.parametrize("period", [None, dpnp.array(180.0)])
```

Comment:
no need to have that as array here, because there will be `dpnp.array(..., sycl_queue=x.sycl_queue)` anyway
```suggestion
@pytest.mark.parametrize("left", [None, -1.0])
@pytest.mark.parametrize("right", [None, 99.0])
@pytest.mark.parametrize("period", [None, 180.0])
```